ui/cypress/integration/dashboard.test.ts

```diff
@@ -0,0 +1,200 @@
+describe('dashboards', () => {
+  beforeEach(() => {
+    cy.deleteDashboards()
+    cy.cutConnections()
+    cy.createConnection()
+    cy.get('@connections').then(connections => {
+      cy.fixture('routes').then(({dashboards}) => {
+        cy.visit(`/sources/${connections[0].id}${dashboards}`)
+      })
+    })
+  })
+
+  it('create, rename and delete a dashboard', () => {
+    // create a dashboard
+    cy.get('button').contains('Create Dashboard').click()
+
+    const newName = 'DashboardQA'
+
+    // rename the dashboard
+    cy.get('.rename-dashboard')
+      .should('have.text', 'Name This Dashboard')
+      .type(`${newName}{enter}`)
+      .should('have.text', newName)
+
+    // delete the dashboard
+    cy.get('@connections').then(connections => {
+      cy.fixture('routes').then(({dashboards}) => {
+        cy.visit(`/sources/${connections[0].id}${dashboards}`)
+      })
+    })
+
+    // DOM Element where the dashboard resides
+    cy.get('.panel-body > table > tbody')
+      .should('exist')
+      .within(() => {
+        // delete button
+        cy.get('.confirm-button--confirmation').click({force: true})
+      })
+      .should('not.exist')
+  })
+})
+
+// TODO: has to be rewritten and sorted into multiple files as it does much more than the test description says (creates flux Query)
```

Comment:
Please remove all of the commented code, or create a defect that requests to fix it (so that it can be panned to get fixed later).

Reply:
Commented code removed as requested.